server/src/main/java/org/elasticsearch/index/snapshots/blobstore/BlobStoreIndexShardSnapshots.java

```diff
@@ -219,6 +226,8 @@ public static BlobStoreIndexShardSnapshots fromXContent(XContentParser parser) t
             token = parser.nextToken();
         }
         Map<String, List<String>> snapshotsMap = new HashMap<>();
+        Map<String, String> historyUUIDs = new HashMap<>();
+        Map<String, Long> sequenceNumbers = new HashMap<>();
```

Comment:
should this be `globalCheckpoints`?

Reply:
Irrelevant now, only passing a single identifier with latest changes.